xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountTransactionsRequestParams.java

```diff
@@ -38,10 +38,70 @@
    *
    * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
    */
+  @Deprecated
   static ImmutableAccountTransactionsRequestParams.Builder builder() {
     return ImmutableAccountTransactionsRequestParams.builder();
   }
 
+  /**
+   * Construct a builder for this class with {@link LedgerIndex} field set.
+   *
+   * @param ledgerIndex An index of type {@link LedgerIndex} to specify a particular ledger.
+   * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
+   */
+  static ImmutableAccountTransactionsRequestParams.Builder builder(LedgerIndex ledgerIndex) {
+    return ImmutableAccountTransactionsRequestParams.builder()
+        .ledgerSpecifier(
+        Optional.of(LedgerSpecifier.of(ledgerIndex))
+    );
+  }
+
+  /**
+   * Construct a builder for this class with {@link LedgerSpecifier} field set.
+   *
+   * @param ledgerSpecifier A specifier of type {@link LedgerSpecifier} to specify a ledger.
+   * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
+   */
+  static ImmutableAccountTransactionsRequestParams.Builder builder(LedgerSpecifier ledgerSpecifier) {
+    return ImmutableAccountTransactionsRequestParams.builder().ledgerSpecifier(Optional.ofNullable(ledgerSpecifier));
+  }
+
+  /**
+   * Construct a builder for this class with ledgerHash field set.
+   *
+   * @param ledgerHash Hash of type {@link Hash256} of a specific ledger.
+   * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
+   */
+  static ImmutableAccountTransactionsRequestParams.Builder builder(Hash256 ledgerHash) {
+    return ImmutableAccountTransactionsRequestParams.builder()
+        .ledgerSpecifier(Optional.of(LedgerSpecifier.of(ledgerHash)));
+  }
+
+  /**
+   * Construct a builder for this class with ledgerIndexRange field set.
+   *
+   * @param ledgerIndexMinimum A lower bound of ledgerIndex of type {@link LedgerIndexBound}.
+   * @param ledgerIndexMaximum An upper bound of ledgerIndex of type {@link LedgerIndexBound}.
+   * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
+   */
+  static ImmutableAccountTransactionsRequestParams.Builder builder(
+      LedgerIndexBound ledgerIndexMinimum,
+      LedgerIndexBound ledgerIndexMaximum) {
+    LedgerIndexBound ledgerIndexMin = ledgerIndexMinimum;
+    LedgerIndexBound ledgerIndexMax = ledgerIndexMaximum;
+
+    if (ledgerIndexMin == null) {
```

Comment:
In this library, `null` is a bug, so we should have overt `Objects.requireNonNull(ledgerIndexMinimum);` and `Objects.requireNonNull(ledgerIndexMaximum);` so that null is not allowed.